core/web/migrate_controller.go

```diff
@@ -123,7 +123,11 @@ func (mc *MigrateController) MigrateJobSpec(c *config.Config, js models.JobSpec)
 	case models.InitiatorCron:
 		return migrateCronJob(js)
 	case models.InitiatorRunLog:
-		return migrateRunLogJob(js)
+		jb, warnDotSep, err := migrateRunLogJob(js)
+		if err == nil && warnDotSep {
+			mc.App.GetLogger().Warn("Run Log job may depend on dot separator")
```

Comment:
@PatrickAlphaC 
1. How can we improve this message to communicate that the default separator changed, and how a node op may need to react? Something about related contracts may need to be replaced or updated?
2. Is it sufficient to log this just on the _node_? I'm thinking this message should be printed locally by the _CLI_ too.

Reply:
Caveat to mention: if they can't use the new default due to keys containing commas, they will have to wait for 1.4 to be released to be able to use a custom `separator` :(

Reply:
Maybe just add `Please use comma separator for versions 1.0.0 -> 1.3.0`